# HG changeset patch
# User Ben Abbott  <bpabbott@mac.com>
# Date 1371972147 -28800
#      Sun Jun 23 15:22:27 2013 +0800
# Node ID 893e9cec62f1f4874be1b7a63110c396d7ca9437
# Parent  12005245b64537c6852384b033cafd199f20bc13
Add missing semicolon to strjoin (missed in 7a97ff5ef42e).

* scripts/strings/strjoin.m: Add missing semicolon.

diff --git a/scripts/strings/strjoin.m b/scripts/strings/strjoin.m
--- a/scripts/strings/strjoin.m
+++ b/scripts/strings/strjoin.m
@@ -73,17 +73,17 @@ function rval = strjoin (cstr, delimiter
   elseif (num > 0 && numel (delimiter) != num - 1)
     error ("strjoin:cellstring_delimiter_mismatch",
       "strjoin: the number of delimiters does not match the number of strings")
   else
     delimiter(end+1) = {""};
   endif
 
   if (num == 0)
-    rval = ""
+    rval = "";
   else
     rval = [[cstr(:).'; delimiter(:).']{:}];
   endif
 
 endfunction
 
 %!assert (strjoin ({"hello"}, "-"), "hello")
 %!assert (strjoin ({"hello", "world"}), "hello world")
